packages/tailwindcss/src/compat/plugin-api.test.ts

```diff
@@ -1,20 +1,30 @@
 import { describe, expect, test, vi } from 'vitest'
-import { compile } from '..'
+import { compile as coreCompile } from '..'
 import plugin from '../plugin'
 import { optimizeCss } from '../test-utils/run'
 import defaultTheme from './default-theme'
-import type { CssInJs, PluginAPI } from './plugin-api'
+import type { CssInJs, Plugin, PluginAPI } from './plugin-api'
 
 const css = String.raw
 
+// TODO: Expand the API changes into the tests below
+function compile(css: string, base: string, { loadPlugin }: { loadPlugin: () => Promise<Plugin> }) {
+  return coreCompile(css, base, {
+    async loadModule(id, base) {
+      let plugin = await loadPlugin()
+      return { module: plugin, base }
+    },
+  })
+}
```

Comment:
I think we should either:
- Make these changes now; OR
- Do it immediately in a follow up PR

Reply:
Ah I forgot about it. 🙈